### ISSUE TITLE
Hardcoded v-prefix in changelog

### ISSUE THREAD

**Issue by afuetterer** [python-semantic-release/python-semantic-release#561]
Hi there, I just recently started using this great library.

I set the tag_format to `{version}` without the v-prefix.
```toml
[tool.semantic_release]
# ...
tag_format = "{version}"
```
In the changelog headers however there is a hardcoded v-prefix.

```markdown
# Changelog

All notable changes to this project will be documented in this file.

## v0.7.1 (2023-02-19)
### Fix
...
```
There is no way to disable this via another setting, right?

Any help would be appreciated. Thank you.

**Comment by bernardcooke53** [comment]
Hey @afuetterer - I don't think there is a way to disable this setting currently, for example it is hardcoded [here](https://github.com/python-semantic-release/python-semantic-release/blob/5cdb05e20f17b12890e1487c42d317dcbadd06c8/semantic_release/changelog/__init__.py#L34).

This should be possible in the upcoming major version - you can try it out yourself with `pip install --upgrade --pre python-semantic-release` :slightly_smiling_face:

**Comment by afuetterer** [comment]
Hi @bernardcooke53, thanks for taking the time to answer me.
That is great news. v8 looks awesome. I will try the pre-release soon.

**Comment by afuetterer** [comment]
Hi @bernardcooke53, I tried the pre-release. What is the best way to give feedback on things I discover? If I understood the configuration correctly, some things don't work as expected.

**Comment by afuetterer** [comment]
For example: Using the pre-release `v8.0.0-alpha.3` seems to not take into account the `tag_format` specified in pyproject.toml, when tagging and releasing?

```
tag_format = "{version}"
```

See my config in https://github.com/afuetterer/minimal-example

It tagged and released v0.1.0, instead of 0.1.0. Or am I doing something wrong here?

**Comment by bernardcooke53** [comment]
Probably best to take other bugs to https://github.com/python-semantic-release/python-semantic-release/issues/487 - thanks for testing it out and for sharing the minimal example, I have used your repo locally to test it out and that should now be fixed in `v8.0.0-alpha.4` :crossed_fingers:

**Comment by afuetterer** [comment]
Excellent, thank you very much for your reply.
I tried alpha.4, the fix works and `minmal-example` gets tagged with the specified tag_format.

I will report other links in #487 as suggested.
